lib/datasampler.rb

```diff
@@ -166,14 +228,18 @@ def export_data(client_list: ['all'], days: 60, anon: true)
     anonymize_clients(clients) if anon
 
     filename = @directory + 'clients.json'
-    File.open(filename, 'w') { |file| file.write(clients.to_json) }
+    File.open(filename, 'w') do |file|
+      file.write(JSON.generate(clients))
+    end
 
     all_projects = Project.where(id: project_ids).naked.all
     anonymize_projects(all_projects) if anon
 
     # write the projects to a file in json fromat
     filename = @directory + 'projects.json'
-    File.open(filename, 'w') { |file| file.write(all_projects.to_json) }
+    File.open(filename, 'w') do |file|
+      file.write(JSON.generate(all_projects))
```

Comment:
From what I've seen, that would only be an issue if we were directly importing the json into sequel. Since we're processing it first, reading it into ruby, then pushing into sequel, we're passing a ruby hash in rather than a string